### PR TITLE
fix: update doc

### PR DESCRIPTION
**Pull request by chuang8511**
Because

- email should be email rather than gmail
- the doc is outdated

This commit

- change title
- update the doc

### CODE REVIEW

File: ai/instill/v0/README.mdx [hunk 1]
```diff
@@ -310,10 +310,3 @@ Generate image from input text prompt and image.
 | Output | ID | Type | Description |
 | :--- | :--- | :--- | :--- |
 | Images | `images` | array[string] | Images |
-
```
Comment (jvallesm) [inline ai/instill/v0/README.mdx:313]: Please fetch the latest `compogen`. These newlines are produced by changes in the `compogen` template to improve its readability (we don't care about readability in `README.mdx`, only on its HTML version, which removes redundant spaces.

**Review by jvallesm — COMMENTED** [review]
LGTM besides minor comment

File: ai/instill/v0/README.mdx (hunk 1 above)
Comment (chuang8511) [reply ai/instill/v0/README.mdx:313]: @jvallesm 
Ah, I forgot to sync with you yesterday.
I did fetch the latest compogen.
But, it seems conflict with pre-commit hook now.

image

Comment (chuang8511) [reply ai/instill/v0/README.mdx:313]: @jvallesm 
I did fetch the latest compogen.
But, it seems conflict with pre-commit hook now.
When I commit, those redundant lines are delete by pre-commit hook.

Comment (chuang8511) [reply ai/instill/v0/README.mdx:313]: Which one should we follow now?

Comment (donch1989) [reply ai/instill/v0/README.mdx]: @chuang8511 @jvallesm 
I add a rules to exclude the pre-commit-hooks for mdx files.

**Pull request merged**